doc/source/theory/dem/dem.rst

```diff
@@ -0,0 +1,182 @@
+    \mathbf{F}_{ij}^t &= -(k_t\mathbf{\delta}_t)-(\eta_t\mathbf{v}_{rt})
+
+Where : 
+
+* :math:`\delta_n` normal overlap;
+* :math:`\delta_t` tangential overlap;
+* :math:`\mathbf{n}_{ij}` contact normal vector;
+* :math:`k_n, k_t` spring constants;
+* :math:`\eta_n, \eta_t` damping model constants;
+* :math:`\mathbf{v}_{rn}` relative velocity in the normal direction;
+* :math:`\mathbf{v}_{tn}` relative velocity in the tangential direction;
+
+
+.. figure:: images/collision_particles.png
+    :width: 400
+    :align: center
+    :alt: particle-particle_collision
+
+    Representation of a typical particle-particle contact. [1]
+
+The contact normal vector :math:`\mathbf{n}_{ij}` is computed as : 
+
+.. math::
+    \mathbf{n}_{ij}=\frac{\mathbf{x}_{j}-\mathbf{x}_{i}}{\left|\mathbf{x}_{j}-\mathbf{x}_{i}\right|}
+
+The normal overlap (:math:`\delta_n`) is the contact distance between the particles i and j. The tangential overlap (:math:`\delta_t`) depends on the contact history and is updated during a contact.
+The normal and tangential overlaps are calculated as follow : 
+
+.. math::
+    \delta_n =& \:R_i + R_j - |\mathbf{x}_{j} - \mathbf{x}_{i}| \\
+    \mathbf{\delta}_{ij}^{t,\text{new}} &= \mathbf{\delta}_{ij}^{t,\text{old}}+\mathbf{v}_{rt}dt
+
+
+The relative velocities calculated to allow updating the tangential overlap are then described by :
+
+.. math::
+    \mathbf{v}_{rn} &= \left(\mathbf{v}_{ij}.\mathbf{n}_{ij}\right)\mathbf{n}_{ij} \\
+    \mathbf{v}_{rt} &= \mathbf{v}_{ij}-\mathbf{v}_{rn} \\
+    \mathbf{v}_{ij} &= \mathbf{v}_i-\mathbf{v}_j+\left(R_i\mathbf{\omega}_i+R_j\mathbf{\omega}_j\right)\times\mathbf{n}_{ij}
+
+
+The linear models used in Lethe-DEM are the following equations :
+
+.. math::
+    k_n &= \frac{16}{15}\sqrt{R_{e}}Y_{e}\left(\frac{15m_{e}V^2}{16\sqrt{R_{e}}Y_{e}}\right)^{0.2} \\
+    \eta_n &= \sqrt{\frac{4m_{e}k_n}{1+\left(\frac{\pi}{\ln{e}}\right)^2}} \\
+    k_t &= k_n \\
+    \eta_t &= \eta_n
+
+
+Nonlinear viscoelastic models used are :
+
+.. math::
+    k_n &= \frac{4}{3}Y_{e}\sqrt{R_{e}\delta_n} \\
+    \eta_n &= -2\sqrt{\frac{5}{6}}\beta\sqrt{S_nm_{e}} \\
+    k_t &= 8G_{e}\sqrt{R_{e}\delta_n} \\
+    \eta_t &= -2\sqrt{\frac{5}{6}}\beta\sqrt{S_tm_{e}}
+
+
+Where : 
+
+* :math:`R_e` effective radius;
+* :math:`Y_e` effective Young's modulus;
+* :math:`m_e` effective mass;
+* :math:`V` characteristic impact velocity;
+* :math:`e` coefficient of restitution;
+* :math:`G_e` effective shear modulus;
+
+The parameters are computed as followed :
+
+.. math::
+    \frac{1}{m_{e}} &= \frac{1}{m_i}+\frac{1}{m_j} \\
+    \frac{1}{R_{e}} &= \frac{1}{R_i}+\frac{1}{R_j} \\
+    \frac{1}{G_{e}} &= \frac{2(2-\nu_i)(1+\nu_i)}{Y_i}+\frac{2(2-\nu_j)(1+\nu_j)}{Y_j} \\
+    \frac{1}{Y_{e}} &= \frac{\left(1-\nu_i^2\right)}{Y_i}+\frac{\left(1-\nu_j^2\right)}{Y_j} \\
+    \beta &= \frac{\ln{e}}{\sqrt{\ln^2{e}+\pi^2}} \\
+    S_n &= 2Y_{e}\sqrt{R_{e}\delta_n} \\
+    S_t &= 8G_{e}\sqrt{R_{e}\delta_n}
```

Comment:
Is your suggestion specifically about those equations or is the DEM documentation overall?

Reply:
I tried something for the spring and damping models, let me know how you like it. Hard to make a table for those parameters since some don't have names (Beta, Sn & St).